roles/openshift_openstack/templates/heat_stack_server.yaml.j2

```diff
@@ -228,7 +228,7 @@ resources:
       metadata:
         group: { get_param: group }
         environment: { get_param: cluster_env }
-        clusterid: { get_param: cluster_id }
+        clusterid: {{ openshift_openstack_clusterid }}
```

Comment:
We should delete the `cluster_id` Heat parameter since we're no longer using it.